easybuild/easyblocks/l/lua.py

```diff
@@ -68,6 +68,9 @@ def configure_step(self, cmd_prefix=''):
             mycflags.append('-DLUA_COMPAT_5_1')
         if LooseVersion(self.version) > LooseVersion('5.2'):
             mycflags.append('-DLUA_COMPAT_5_2')
+        if LooseVersion(self.version) > LooseVersion('5.3'):
+            mycflags.append(os.getenv('CFLAGS', ''))
```

Comment:
why is this behind an if? It should be used in general?

Reply:
+1

Reply:
fixed in dda269b and re-tested with all existing Lua easyconfigs, works like a charm